aries-site/src/examples/templates/dashboards/ThreeColumnDashboard.js

```diff
@@ -1,21 +1,30 @@
 import { useContext, useState } from 'react';
-import { Grid, Main, ResponsiveContext } from 'grommet';
 import {
-  AppContainer,
-  PageContainer,
-  PageContainerContext,
-} from '../page-layouts/components';
+  Grid,
+  Main,
+  ResponsiveContext,
+  Page,
+  PageContent,
+  Header,
+  Heading,
+} from 'grommet';
+import { AppContainer } from '../page-layouts/components';
 import { ContentArea } from '../page-layouts/anatomy/components';
-import { PageHeader } from './components';
 import { CostByMonth, CostByService, CostByYear, RulesAudit } from './content';
 
 export const ThreeColumnDashboard = () => (
   <AppContainer background="background-back">
     <ContentArea title="Global Header" />
-    <PageContainer>
-      <PageHeader title="Dashboard" />
```

Comment:
Any reason we are removing PageHeader as opposed to adjusting PageHeader.js to align with the needs for these examples?

Reply:
It felt like having a separate PageHeader file wasn't necessary all it would be is
```
<Header>
  <Heading>
    Page Title
  </Heading>
</Header>
```

Reply:
There are a couple of other considerations:
1. We want to move towards building a browse-able library of component templates whether they are at the organism level or higher
2. For maintenance and consistency, we want to move away from one-off implementations which tend to drift over time. Instead we'd like to move towards component re-use to enforce consistency, even if they are trivial components at this time, we can build atop of them as needs and usage evolves. Heading this direction becomes the basis for which feeds the template library ☝️ .
3. There is also an educational component where we are trying to focus on presenting a particular concept in the example. Sometimes the focus on that concept is diffused if there are too many elements or moving pieces. It becomes harder to see the forrest (big picture) through the trees (extra markup, inline logic, etc.). Sometimes abstracting helps reduce the cognitive load and is a more effective educational mechanism.

Reply:
Gotcha, thanks for the explanation! I will move the Header back into the PageHeader file